.github/workflows/macos-26-matrix.yml

```
@@ -0,0 +1,285 @@
+            os: macos-26
+            mpi: mpich
+            shared_libs: ON
+            static_libs: ON
+            cpp: ON
+            fortran: ON
+            java: OFF
+            hl: ON
+            threadsafe: OFF
+            parallel: ON
+            tools: ON
+            tests: ON
+            examples: ON
+            zlib: ON
+            szip: ON
+
+          # Serial build with thread-safety
+          - name: "macOS-26 Serial Threadsafe"
+            os: macos-26
+            mpi: none
+            shared_libs: ON
+            static_libs: ON
+            cpp: ON
+            fortran: OFF
+            java: OFF
+            hl: OFF
+            threadsafe: ON
+            parallel: OFF
+            tools: ON
+            tests: ON
+            examples: ON
+            zlib: ON
+            szip: ON
+
+          # Minimal configuration
+          - name: "macOS-26 Minimal Config"
+            os: macos-26
+            mpi: none
+            shared_libs: ON
+            static_libs: OFF
+            cpp: OFF
+            fortran: OFF
+            java: OFF
+            hl: OFF
+            threadsafe: OFF
+            parallel: OFF
+            tools: OFF
+            tests: ON
+            examples: OFF
+            zlib: OFF
+            szip: OFF
+
+          # Plugin support with OpenMPI
+          - name: "macOS-26 Plugins OpenMPI"
+            os: macos-26
+            mpi: openmpi
+            shared_libs: ON
+            static_libs: OFF
+            cpp: OFF
+            fortran: OFF
+            java: OFF
+            hl: ON
+            threadsafe: OFF
+            parallel: ON
+            tools: ON
+            tests: ON
+            examples: ON
+            zlib: ON
+            szip: ON
+            plugins: ON
+
+          # No High Level API with MPICH
+          - name: "macOS-26 No HL MPICH"
+            os: macos-26
+            mpi: mpich
+            shared_libs: ON
+            static_libs: ON
+            cpp: ON
+            fortran: OFF
+            java: OFF
+            hl: OFF
+            threadsafe: OFF
+            parallel: ON
+            tools: ON
+            tests: ON
+            examples: ON
+            zlib: ON
+            szip: ON
+
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, 'skip-ci')"
+
+    steps:
+    - name: Install Dependencies
+      run: |
+        # Install base dependencies
+        brew install autoconf automake libtool
+
+        # Install compression libraries
+        brew install zlib
+        if [[ "${{ matrix.szip }}" == "ON" ]]; then
+          brew install libaec
+        fi
+
+        # Install MPI libraries
+        if [[ "${{ matrix.mpi }}" == "openmpi" ]]; then
+          brew install openmpi
+        elif [[ "${{ matrix.mpi }}" == "mpich" ]]; then
+          brew install mpich
+        fi
+
+
+    - name: Get Sources
+      uses: actions/checkout@v5
+
+    - name: Configure CMake
```

```suggestion
    - name: Configure
```
PR #5813 Removed "CMake" from CI names, descriptions, etc.